src/main/java/core/basesyntax/supplier/FigureSupplier.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax.supplier;
+
+import core.basesyntax.model.Circle;
+import core.basesyntax.model.Figure;
+import core.basesyntax.model.IsoscelesTrapezoid;
+import core.basesyntax.model.Rectangle;
+import core.basesyntax.model.RightTriangle;
+import core.basesyntax.model.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Random random = new Random();
+
+    public Figure getRandomFigure() {
+        int indexFigure = random.nextInt(4);
```

Comment:
```suggestion
        int indexFigure = random.nextInt(FIGURE_NUMBER);
```
lets make constant for magic number 4